releasenotes/notes/11130.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: release-notes/v2
+kind: feature
+area: traffic-management
+
+# issue is a list of GitHub issues resolved in this note.
+issue:
+  - https://github.com/istio/istio/issues/11130
```

Comment:
This duplicates the issue entry.

Reply:
I saw this type of double entry in another release note, so I wasn't sure if it was necessary or not. I've now removed this.

Reply:
> I'm not sure it needs an upgrade note as it's off by default but we do want the contents of what is there somewhere at least, so users understand the risks of turning it on. That could be a proper doc possible, not sure - will leave to the docs folks to decide
> […](#)
> On Wed, Jul 22, 2020, 6:37 AM Brian Avery ***@***.***> wrote: ***@***.**** commented on this pull request. ------------------------------ In releasenotes/notes/11130.yaml <[#25739 (comment)](https://github.com/istio/istio/pull/25739#discussion_r458796925)>: > +kind: feature +area: traffic-management + +# issue is a list of GitHub issues resolved in this note. +issue: + - #11130 + - 11130 + +releaseNotes: | + *Added* config option `values.global.proxy.holdApplicationUntilProxyStarts`, + which causes the sidecar injector to inject the sidecar at the start of the + pod's container list and configures it to block the start of all other + containers until the proxy is ready + +upgradeNotes: + - title: Impact on application start time This change doesn't break backwards compatibility with 1.6 does it? If not, I'm not sure it needs an upgrade note at all. — You are receiving this because your review was requested. Reply to this email directly, view it on GitHub <[#25739 (comment)](https://github.com/istio/istio/pull/25739#discussion_r458796925)>, or unsubscribe <[github.com/notifications/unsubscribe-auth/AAEYGXPVWLW7WUAYPNOLFITR43TQDANCNFSM4PEPN4KA](https://github.com/notifications/unsubscribe-auth/AAEYGXPVWLW7WUAYPNOLFITR43TQDANCNFSM4PEPN4KA)> .

I agree. @ericvn @frankbu Any opinions on where the upgrade notes content should go?